createdisk-library.sh

```diff
@@ -58,29 +58,31 @@ EOF
 
 function create_qemu_image {
     local destDir=$1
+    local base=$2
+    local overlay=$3
 
     if [ -f /var/lib/libvirt/images/${VM_PREFIX}-master-0 ]; then
-      sudo cp /var/lib/libvirt/images/${VM_PREFIX}-master-0 $destDir
-      sudo cp /var/lib/libvirt/images/${VM_PREFIX}-base $destDir
+      sudo cp /var/lib/libvirt/images/${overlay} ${destDir}
+      sudo cp /var/lib/libvirt/images/${base} ${destDir}
     else
-      sudo cp /var/lib/libvirt/openshift-images/${VM_PREFIX}/${VM_PREFIX}-master-0 $destDir
-      sudo cp /var/lib/libvirt/openshift-images/${VM_PREFIX}/${VM_PREFIX}-base $destDir
+      sudo cp /var/lib/libvirt/openshift-images/${VM_PREFIX}/${overlay} ${destDir}
+      sudo cp /var/lib/libvirt/openshift-images/${VM_PREFIX}/${base} ${destDir}
     fi
 
-    sudo chown $USER:$USER -R $destDir
-    ${QEMU_IMG} rebase -f qcow2 -F qcow2 -b ${VM_PREFIX}-base $destDir/${VM_PREFIX}-master-0
-    ${QEMU_IMG} commit $destDir/${VM_PREFIX}-master-0
+    sudo chown $USER:$USER -R ${destDir}
+    ${QEMU_IMG} rebase -f qcow2 -F qcow2 -b ${base} ${destDir}/${overlay}
+    ${QEMU_IMG} commit ${destDir}/${overlay}
 
-    sparsify $destDir ${VM_PREFIX}-base ${CRC_VM_NAME}.qcow2
+    sparsify ${destDir} ${base} ${overlay}
```

Comment:
@cfergeau here we are sparsify `base => crc.qcow2` not `base => crc-master-0` . I can see you added `mv overlay -> vm_name.qcow2` in createdisk.sh which might be not required since we source this script to `createdisk.sh` which already have `CRC_VM_NAME` variable. Or you have something else in your mind.

Reply:
In case someone reads this in the future :)
In the openshift branch, we indeed have 3 distinct filenames, ${base} ${overlay} and ${CRC_VM_NAME}.qcow2, and create_qemu_image deletes ${base} and $overlay} once it's done creating ${CRC_VM_NAME}.qcow2
However in the podman branch,  ${overlay} is the same as ${CRC_VM_NAME}.qcow2, so we only have 2 distinct files. At the end of create_qemu_image, we only delete ${base} in the podman branch.
So it's better to not use ${CRC_VM_NAME} here, and to keep the mv, otherwise it's harder to hav esimilar code between both branches.